assets/app/view/create_game.rb

```diff
@@ -408,6 +421,13 @@ def submit
         game_data[:settings][:seed] = game_params[:seed] if game_params[:seed]
       end
 
+      checked_options = Engine.meta_by_title(game_data[:title])
+                          .check_options(game_data[:settings][:optional_rules],
+                                         game_data[:min_players], game_data[:max_players])
```

Comment:
Why doesn't this need a .respond_to? guard clause?

Reply:
On the branch where I'm fixing the 1822PNW variants I added it to the base meta class, I tried splitting it out from this branch but that should be here too. And I guess with that method I can get rid of the other `.respond_to(:check_options)` here. What do you think? b89fe89

Reply:
Well, if check_options is now in the base class, then you should definitely get rid of all the repond_to guards.